x-pack/legacy/plugins/integrations_manager/public/utils/remove_relative_path.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const removeRelativePath = (relativePath: string): string | null => {
```

Comment:
Returning `/a/c` from `../../../a/b/../c` seems like it would also be incorrect.  I'm not sure how we'd support that type of path or if we could? I'd need to know the name of that relative directory between the two named directories.